sdk/python/kfp/components/types/type_utils_test.py

```diff
@@ -640,5 +581,282 @@ def test_robustness_to_literals(
         self.assertEquals(res, expected_val)
 
 
+class TestTypeChecking(parameterized.TestCase):
+
+    @parameterized.parameters(
+        # param True
```

Comment:
nit: JFYI, alternatively the comments can be put in the `name` field of a test case, so that when some case failed, it could be more easily to identify which case failed from the error log.

Reply:
Ack. This is good to know!